Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.py

```diff
@@ -0,0 +1,321 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+import botocore.exceptions
```

Comment:
Is this used?

Reply:
Yes botocore is essential for AWS authentication and other methods